server/NetPacksServer.cpp

```diff
@@ -22,145 +22,145 @@
 #include "../lib/spells/ISpellMechanics.h"
 
 
-#define PLAYER_OWNS(id) (gh->getPlayerAt(c)==gh->getOwner(id))
-#define ERROR_AND_RETURN												\
-	do { if(c) {														\
-			SystemMessage temp_message("You are not allowed to perform this action!"); \
-			boost::unique_lock<boost::mutex> lock(*c->wmx);				\
-			*c << &temp_message;										\
-		}																\
-		logNetwork->error("Player is not allowed to perform this action!");		\
-		return false;} while(0)
-
-#define WRONG_PLAYER_MSG(expectedplayer) do {std::ostringstream oss;\
-			oss << "You were identified as player " << gh->getPlayerAt(c) << " while expecting " << expectedplayer;\
-			logNetwork->error(oss.str()); \
-			if(c) { SystemMessage temp_message(oss.str()); boost::unique_lock<boost::mutex> lock(*c->wmx); *c << &temp_message; } } while(0)
-
-#define ERROR_IF_NOT_OWNS(id)	do{if(!PLAYER_OWNS(id)){WRONG_PLAYER_MSG(gh->getOwner(id)); ERROR_AND_RETURN; }}while(0)
-#define ERROR_IF_NOT(player)	do{if(player != gh->getPlayerAt(c)){WRONG_PLAYER_MSG(player); ERROR_AND_RETURN; }}while(0)
+#define PLAYER_OWNS(id) (gh->getPlayerAt(c) == gh->getOwner(id))
+#define ERROR_AND_RETURN                                                                                                \

```

Comment:
It would be great if uncrustify could format macro body that:
a) use rules as for formatting code
b) align trailing `\` on the same column